src/Equinox.MessageDb/MessageDbClient.fs

```diff
@@ -1,25 +1,23 @@
-namespace Equinox.MessageDb
+namespace Equinox.MessageDb.Core
 
+open FsCodec
+open FsCodec.Core
+open Npgsql
+open NpgsqlTypes
 open System
 open System.Data.Common
 open System.Text.Json
 open System.Threading
 open System.Threading.Tasks
-open FsCodec
-open FsCodec.Core
-open Npgsql
-open NpgsqlTypes
 
 [<RequireQualifiedAccess; NoEquality; NoComparison>]
 type MdbSyncResult = Written of int64 | ConflictUnknown
 type private Format = ReadOnlyMemory<byte>
 
-type MessageDbClient(source: CancellationToken -> Task<NpgsqlConnection>) =
+type MessageDbClient(createConnection: CancellationToken -> Task<NpgsqlConnection>) =
```

Comment:
maybe this can be `type internal` (depends whether tests use it)?

Reply:
Not quite sure how to achieve that. If I make it internal the compiler complains because it's on the connection class, which is used by the context, etc.

Reply:
Oh wait, did you mean `type X internal (...)` 🤦. I'll do that

Reply:
Had not realised it's role; it's obviously fine for it to be public (but hidden in the Core sub-namespace) in that case
minimising visibility of ctors is always good - if anyone ever needs it public, that can be done if needed